File: backend/Logger-Service/Logger-Service/LoggerService/Models/LogMessage.cs
namespace LoggerService.Models
{
    public class LogMessage
    {
        public string ServiceName { get; set; }

        public string UserRole { get; set; }

        public long? UserId { get; set; }

        public string Message { get; set; }

        public string LogLevel {  get; set; }

        public DateTime Timestamp { get; set; }

        public string RequestId { get; set; }
    }
}

File: backend/payment-service/src/main/java/com/cdac/dto/LogMessage.java
package com.cdac.dto;

import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LogMessage {

    private String serviceName;

    private String userRole;

    private Long userId;

    private String logLevel;

    private String message;

    private LocalDateTime timestamp;

    private String requestId;
}